src/Domain/Regulation/Repository/LocationRepositoryInterface.php

```diff
@@ -17,7 +17,7 @@ public function findOneByUuid(string $uuid): ?Location;
     public function findAllForMapAsGeoJSON(
         bool $includePermanentRegulations = false,
         bool $includeTemporaryRegulations = false,
-        bool $includeUpcomingRegulations = false,
-        bool $includePastRegulations = false,
+        bool $includeMeasureTypeNoEntry = false,
```

Comment:
J'ai l'impression qu'on pourrait en profiter pour restructurer la façon dont on gère les "choix" dans les filtres

Si à l'avenir on implémente de nouveaux types de restrictions dans DiaLog, il faudra revenir ici et rajouter des paramètres dans les fonctions, les DTO, etc.

En fait je pense que chaque groupe de checkboxes sur la maquette correspond à _un_ filtre qui serait de type "choix multiple".

Ça correspond au `ChoiceType` de Symfony avec l'option `'multiple' => true`, il y a un exemple dans le champ `critairTypes` de `VehicleSetFormType`

De cette façon le filtre est représenté par un array qui contient les valeurs auxquelles on souhaite restreindre les résultats. Par ex si seulement "Circulation interdite" est coché on aurait la valeur `$measureTypes = [MeasureTypeEnum::NO_ENTRY->value]`

Donc on utiliserait un array comme ceci:

```suggestion
        array $measureTypes,
```